cvat/apps/engine/serializers.py

```diff
@@ -391,11 +393,24 @@ def validate_chunk_size(self, value):
             raise serializers.ValidationError('Chunk size must be a positive integer')
         return value
 
+    def validate_filename_pattern(self, value):
+        supported_wildcards = set(["*", "[", "]", "."])
```

Comment:
If I understand correctly that the dot is for 1 any character, then it probably should be changed to `?` in wildcards and then changed to `.` for regex.